apis/infix-en_GB/atrium-api-infix-en_GB-common/src/test/kotlin/ch/tutteli/atrium/api/infix/en_GB/samples/deprecated/MapEntryAssertionSamples.kt

```diff
@@ -0,0 +1,60 @@
+package ch.tutteli.atrium.api.infix.en_GB.samples.deprecated
+
+import ch.tutteli.atrium.api.infix.en_GB.*
+import ch.tutteli.atrium.api.infix.en_GB.samples.fails
+import ch.tutteli.atrium.api.verbs.internal.expect
+import kotlin.test.Test
+
+class MapEntryAssertionSamples {
+
+    @Test
+    fun isKeyValue() {
```

Comment:
you forgot to add the `@sample` in the KDoc in mapEntryExpectations.kt

Reply:
Thanks for adding this documentation in addition (next to MapExpectationSamples). You need to add the `@sample` tag to the functions in mapEntryAssertions.kt (as you need to do it for mapEntryExpectations.kt) if you want that the user of Atrium actually sees it in the docs